tasks/scrapeData/dedupData.js

```diff
@@ -0,0 +1,113 @@
+import * as geography from '../../lib/geography.js';
+
+const numericalValues = ['cases', 'tested', 'recovered', 'deaths', 'active'];
+
+/*
+  Returns a report if the crosscheck fails, or false if the two sets have identical data
+*/
+function crosscheck(a, b) {
+  const crosscheckReport = {};
+  let failed = false;
+  for (const prop of numericalValues) {
+    if (a[prop] !== b[prop]) {
+      crosscheckReport[prop] = [a[prop], b[prop]];
+      failed = true;
+    }
+  }
+  return failed ? crosscheckReport : false;
+}
+
+/*
+  Remove "private" object properties
+*/
+function removePrivate(data) {
```

Comment:
@qgolsteyn can we share these? They could live in `lib/transform.js`

Reply:
Would prefer keeping them to this file as they are not used anywhere else for now. Seems more of a private method.